<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWidget</class>
 <widget class="QWidget" name="ConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DropN900 - Log</string>
  </property>
  <property name="styleSheet">
   <string>QWidget#ConsoleWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(6, 6, 6, 255), stop:1 rgba(45, 45, 45, 255));
}

QLabel#title_download_folder, #title_authentication, #title_sync {
	color: #0099FF;
	font-size: 16pt;
	font-weight: bold;
}

QPlainTextEdit#text_area {
	background-color: white;
	border: 2px solid rgba(255,255,255,200);
	border-radius: 10px;
	font-size: 10pt;
}

QPushButton {
	min-height: 65px;
	border-radius: 10px;
	border: 2px solid rgba(255,255,255,200);
	color: rgb(230, 230, 230);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(16, 16, 16, 255), stop:0.5 rgba(55, 55, 55, 210), stop:1 rgba(36, 36, 36, 255));
	font: Arial;
	font-size: 19pt;
}

QPushButton#button_save{
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
	border-right: 0px;
}

QPushButton#button_back {
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
}

QPushButton:hover {
	color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(59, 59, 59, 255), stop:0.5 rgba(82, 82, 82, 255), stop:1 rgba(68, 68, 68, 255));
	color: #0099FF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="text_area">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="font">
        <font>
         <pointsize>19</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>SAVE LOG TO FILE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_back">
       <property name="font">
        <font>
         <pointsize>19</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>RETURN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
